README.md

```diff
@@ -1 +1,8 @@
-# create202404
\ No newline at end of file
+# create202404
+Some description that doesnt mean anything.
+
+## Section 2
+How to enable something that need to be enabled without the need to enable it.
+
```

Comment:
The order of the sections in the README seems incorrect. Section 1 should logically precede Section 2 for better readability and structure.
```suggestion
# create202404
Some description that doesnt mean anything.

## Section 1
Section that should be before #section2 but for some reason is after

## Section 2
How to enable something that need to be enabled without the need to enable it.
```